docs/05-Navigation Interception.md

```diff
@@ -0,0 +1,16 @@
+## Navigation Interception
+
+By default, PalindromDOM intercepts browser navigation and sends an AJAX request to the server for the next page's partial/model. This results in a more efficient and faster navigation.
+
+PalindromDOM only intercepts links that are part of your application and doesn't intercept external links or links that have a `target` set to anything except `self`.
+
+### How to disable interception
+
+Sometimes, you need to disable interception even for links within the application. And as mentioned above, If you set any `target` that doesn't equal `self` to your link, the navigation from that anchor will not be intercepted.
+
+Example:
+
+```html
+<a href="internalLink.html" target="hello">Go to internal link></a>
+```
+... even if `internalLink.html` is part of your application, PalindromDOM will not intercept the navigation and the browser will issue a full HTTP request to `internalLink.html`.
```

Comment:
As the value of the `target` attribute, you might consider one of the following keywords that have a special meaning in HTML:

- `_top` - load the URL into the current browser tab or window
- `_blank` - load the URL into a new browser tab or window

You can find more information on the `target` attribute at [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).